lib/EditableList/processBadResponse.js

```diff
@@ -0,0 +1,69 @@
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { errorCodes } from './constants';
+
+const getValues = (parameters) => {
+  return parameters.reduce((result, { key, value }) => {
+    result[key] = value;
+
+    return result;
+  }, {});
+};
+
+const getErrorMessage = (errorCode, values = {}) => {
+  return (
+    <FormattedMessage
+      key={errorCode}
+      id={`stripes-smart-components.error.${errorCode}`}
+      values={values}
+    />
+  );
+};
+
+const handleError = (errorMessages, message, values) => {
+  if (values && values.fieldLabel) {
+    if (errorMessages.fieldErrors[values.fieldLabel]) {
+      errorMessages.fieldErrors[values.fieldLabel].push(message);
```

Comment:
This function works just fine as is, but, would it be better to return an updated copy of `errorMessages` rather than modifying it in place? Mutating arguments feels icky.

Reply:
Done.